docs/installing-upgrading.md

```diff
@@ -2,7 +2,7 @@ This is a short guide for running REMS as an uberjar. We also provide a [Dockerf
 
 # Installing REMS
 
-1. Set up a PostgreSQL database.
+1. Set up a PostgreSQL database and create the data model with e.g. `migrate` command.
```

Comment:
perhaps running `migrate` should be the second-to-last step? you need the rems.jar and the configuration file.

Reply:
or you could consider combining these instructions with the Upgrading REMS section below

Reply:
Maybe this shouldn't be updated now. Looks like we could have a slightly simplified docker compose setup soon or at least we could combine the instructions here?

Reply:
yeah let's not update this now